app/src/main/java/fr/free/nrw/commons/data/models/Bookmark.kt

```diff
@@ -1,4 +1,4 @@
-package fr.free.nrw.commons.bookmarks
+package fr.free.nrw.commons.data.models
```

Comment:
Hi @ben-mathu thanks for your PR, could you make a model subfolder for every package and move relevant files to it, see customselector (https://github.com/commons-app/apps-android-commons/tree/master/app/src/main/java/fr/free/nrw/commons/customselector) for example

It will make it easy for other developers to find this file when needed.

Reply:
Okay. Should the subfolders be inside the `data.models` package

Reply:
For instance `app/src/main/java/fr/free/nrw/commons/data/models/Bookmark.kt` should be moved to `app/src/main/java/fr/free/nrw/commons/bookmarks/model/Bookmark.kt`

Reply:
I thought the request was to put all models in a single model package

Reply:
Unless you mean `app/src/main/java/fr/free/nrw/commons/data/models/bookmark/Bookmark.kt`

Reply:
The jacoco file is set to use `model` instead of `data/models` so it would be easier this way :)

https://github.com/commons-app/apps-android-commons/blob/master/jacoco.gradle

```
// model
'**/**/model/*',
```

Reply:
but I thought since, for example in this instance, Bookmark.kt was the only model file in the `app/src/main/java/fr/free/nrw/commons/bookmarks` package, so that is why I added it to `app/src/main/java/fr/free/nrw/commons/data/models/` root package. the others models have a specific package name.

Reply:
No need to make any changes to jacoco file. Simply create a `model` folder inside `bookmarks` package and move the `Bookmark.kt` into it. You can do the same for your other PRs later :)

Reply:
@ben-mathu Let us know if you are still working on this PR :)

Reply:
Yeah I am working on it, sorry for the delay.

Reply:
@ben-mathu Thanks for letting us know, feel free to take your time. Don't hesitate to ask for any clarification you need :)